PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Core/Models/Users/AppUser.cs

```diff
@@ -4,6 +4,7 @@ namespace PrivateHospitals.Core.Models.Users;
 
 public class AppUser : IdentityUser
 {
+    public string? ExternalId { get; set; }
```

Comment:
We agreed we should support search by this fields. Do u want us to fullscan Users table?